src/shillelagh/backends/apsw/vt.py

```diff
@@ -334,16 +332,20 @@ def get_create_table(self, tablename: str) -> str:
         )
         return f'CREATE TABLE "{tablename}" ({formatted_columns})'
 
-    def BestIndex(  # pylint: disable=too-many-locals
+    def _build_index(  # pylint: disable=too-many-locals
         self,
         constraints: List[Tuple[int, SQLiteConstraint]],
         orderbys: List[Tuple[int, bool]],
-    ) -> Tuple[List[Constraint], int, str, bool, float]:
+    ) -> Tuple[
```

Comment:
At some point you might want to consider a dataclass, `TypedDict`, `namedtuple`, etc for readability